src/honeysql_postgres/format.cljc

```diff
@@ -129,7 +129,10 @@
   (sqlf/space-join (format-upsert-clause upsert)))
 
 (defmethod format-clause :returning [[_ fields] _]
-  (str "RETURNING " (sqlf/comma-join (map sqlf/to-sql fields))))
+  (->> (flatten fields)
+       (map sqlf/to-sql)
+       (sqlf/comma-join)
+       (str "RETURNING ")))
```

Comment:
Can we use 
```clojure 
(format "RETURNING %s")
``` 

here instead? 

I find `str` slightly harder to understand in the context of a threading macro.

Reply:
ah, never-mind, I guess, `format` behaves weird with `nil`

```clojure
user=> (format "asd %s" nil)
"asd null"
```